lib/adapters/mongodb/index.js

```diff
@@ -78,6 +79,10 @@ module.exports = function (mongodbUrl, options) {
             .then((created) => {
                 return converters.toJsonApi(created.toObject())
             })
+            .catch((err) => {
+                if (err.code === 11000) throw Boom.conflict(err.errmsg)
+                throw Boom.badImplementation(err)
```

Comment:
That's a good point. I'll convert the MongoDB error a little later in the response.